### PR TITLE
Haiku-ize BeMDI a bit

### PR DESCRIPTION
**Pull request by puckipedia**
For http://www.google-melange.com/gci/task/view/google/gci2014/5909178845495296

### CODE REVIEW

File: source/MDIViewMenu.cpp [hunk 1]
```diff
@@ -31,6 +31,8 @@
 #include "MDIClientView.h"
 #include "MDIViewMenu.h"
 
+using namespace std;
```
Comment (waddlesplash) [inline source/MDIViewMenu.cpp]: Is this really necessary?

File: source/ZOrderedView.cpp [hunk 2]
```diff
@@ -30,6 +30,8 @@
 #include "Color.h"
 #include "ZOrderedView.h"
 
+using namespace std;
```
Comment (waddlesplash) [inline source/ZOrderedView.cpp]: Same.

File: source/MDIViewMenu.cpp (hunk 1 above)
Comment (puckipedia) [inline source/MDIViewMenu.cpp]: Yes. Try removing it :P

Comment (waddlesplash) [inline source/MDIViewMenu.cpp]: Can you adjust the code so it isn't necessary?

**Pull request merged**